lib/rules/unit-disallowed-list/README.md

```diff
@@ -157,3 +157,27 @@ The following patterns are considered problems:
 ```css
 @media print and (max-resolution: 100dpi) {}
 ```
+
+### `ignoreFunctions: ["function", "/regex/", /regex/]|"function"|"/regex/"|/regex/`
+
+Ignore units that are inside of the specified functions.
+
+For example, with `["px"]`.
+
+Given:
+
+```json
+"ignoreFunctions": ["calc", "/^translate/"]
+```
+
+The following patterns are _not_ considered problems:
+
+<!-- prettier-ignore -->
+```css
+a { margin: calc(50% - 100px) }
+```
+
+<!-- prettier-ignore -->
+```css
+a { transform: translateX(50% - 100px) }
```

Comment:
[suggest] Calculation in [`translateX()`](https://developer.mozilla.org/en-US/docs/Web/CSS/transform-function/translateX) seems invalid. Can you please fix it?

```suggestion
a { transform: translateX(100px) }
```